doc/configuration.rst

```diff
@@ -2539,6 +2563,49 @@ Arguments:
  Tools:
   - sunxi-fel: Path to the 'sunxi-fel' tool (default is 'sunxi-fel')
 
+TegraUSBDriver
+~~~~~~~~~~~~~~
+A :any:`TegraUSBDriver` is used to upload an image into a device in the
+*Nvidia Tegra loader state*. This is useful to bootstrap a bootloader onto a
+device.
+
+The load happens in a single stage, with U-Boot proper sent along with a
+*BCT* (Binary Control Tool) configuration file which contains memory timings,
+etc.
```

Comment:
Can this driver be used with U-Boot only? Otherwise it should be worded in a generic way.

Reply:
I'm not sure there is any other loader, but I'll make the U-Boot mention an example.